autopush/truss/handler.py

```diff
@@ -0,0 +1,234 @@
+import time
+import uuid
+import json
+
+from boto.dynamodb2.exceptions import ItemNotFound
+from twisted.internet.threads import deferToThread
+from twisted.internet.defer import Deferred
+
+from autopush.base import (
+    BaseWebHandler,
+    threaded_validate,
+)
+from autopush.db import (
+    generate_last_connect,
+    hasher,
+)
+from autopush.utils import generate_hash, ms_time
+from autopush.truss.validation import (
+    TrussSchema,
+)
+
+
+class RegistrationHandler(BaseWebHandler):
+    cors_methods = "POST,PUT,DELETE"
+
+    #############################################################
+    #                    Cyclone HTTP Methods
+    #############################################################
+    @threaded_validate(TrussSchema)
+    def post(self, truss_data):
+        """HTTP POST
+
+        Endpoint generation and optionally router type/data registration.
+
+        :param truss_data: Dictionary of validated values.
```

Comment:
Agreed. I should just mark the args as "ignored" since we don't use them anyway.